src/connectionController.ts

```diff
@@ -327,11 +327,14 @@ export default class ConnectionController {
     this._connectingConnectionId = connectionId;
     this.eventEmitter.emit(DataServiceEventTypes.CONNECTIONS_DID_CHANGE);
 
+    const prevConnectionId = this._currentConnectionId;
+    const nextConnectionName = this.getSavedConnectionName(connectionId);
+
     if (this._activeDataService) {
       log.info('Disconnecting from the previous connection...', {
         connectionId: this._currentConnectionId,
       });
-      await this.disconnect();
+      await this.disconnect(true);
```

Comment:
```suggestion
      await this.disconnect({ quiet: true });
```
I would prefer having an object as an argument here, so the intention is easier readable from the code.